### ISSUE TITLE
Very hard to see the search result.

### ISSUE THREAD

**Issue by roachsinai** [sainnhe/gruvbox-material#32]
![image](https://user-images.githubusercontent.com/9500049/79074585-d9d08100-7d1f-11ea-956c-3467e501facc.png)

I've `set hlsearch`, and can't find the search result of a search pattern(only the cword could see).

My gruvbox-material setting is:

```
let g:gruvbox_material_background = 'soft'
set background=dark
colorscheme gruvbox-material
```

**Comment by sainnhe** [comment]
Should be fixed via 5444d67.

**Comment by roachsinai** [comment]
Yeah, thanks a lot!
